### ISSUE TITLE
Deleting files fails

### ISSUE THREAD

**Issue by Kyrodan** [ilcgmbh/gulp-nwabap-ui5uploader#6]
If files need to get deleted on NW server, a 404-Result-Code is returned.

**Comment by cassioiks** [comment]
+1: need this!

**Comment by Kyrodan** [comment]
Hi @cassioiks,

I was not able to find the bug, yet, but I updated `gulp-nwabap-ui5uploader` to v0.3.0. This version has imroved logging and error output handling. Please use this version and report your concrete errors in this issue. So I hope this logging helps us to find the bug - maybe it is a bug in NW-Server (in the ABAP Devlopment Tools) and not `gulp-nwabap-ui5uploader`.

Regrads,
Daniel

**Comment by cassioiks** [comment]
Thanks @Kyrodan ,

I don't think it's an issue with the NW server, since all the operations done with the Eclipse ABAP tools work perfectly (delete, update, create).
I will update to version 0.3.0 and do some testing with that. If I find anything I'll report back on this thread.

Regards

**Comment by cassioiks** [comment]
@Kyrodan ,

I tried it now and the error persists.
Steps:
1) Generated Component-preload.js locally
2) Uploaded all the files to NW server
3) Deleted Component-preload.js locally
4) Try to upload again
5) Error happens:

```
Request-Method: delete
Error: got 404 response
Details:
No suitable resource found
```

**Comment by Kyrodan** [comment]
Ok @cassioiks,

I can reproduce a similar example and investigated it. Currently I have no real solution. So I have to trace, what Eclipse ABAP Developer Tools do different than I do. Please be patient!

Regards,
Daniel

**Comment by brooklynb7** [comment]
Hi, I guess I find the issue. You can locate to the line 648 of the file ./lib/filestore.js
The current code is:
sUrl += '&corrNr=' + encodeURIComponent(me._oOptions.ui5.transportno);
But it should be 
sUrl += '?corrNr=' + encodeURIComponent(me._oOptions.ui5.transportno);

And I tried this correction and it works now.
Hope this can help on this bug.
Thanks.

**Comment by Kyrodan** [comment]
Hi @brooklynb7,

thanks for digging into it. Are you sure this fixes this issue? I already found this some time ago in my local working copy but the delete-issue is still present. 

But maybe it is not that simple: I get several different error messages for different delete actions in different call orders and I'm unsure whether it is the same bug. 

I will merge your PR and release a fixed version - than we can look forward.

**Comment by Kyrodan** [comment]
Published 0.4.1
